.test-infra/tools/README.md

```diff
@@ -0,0 +1,61 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Thanks a lot!
Let's advertise this script on https://s.apache.org/beam-python-dev-wiki and/or Jenkins portion of the Wiki once this is merged and document what would it take to install a new Python version using this script.